app/gen-server/lib/homedb/UsersManager.ts

```diff
@@ -435,6 +435,12 @@ export class UsersManager {
         user.options = {...(user.options ?? {}), authSubject: userOptions.authSubject};
         needUpdate = true;
       }
+      // We might want to store extra information returned by the identity provider
+      if (options.profile?.extra) {
+        // Update already existing user options
+        user.options = {...user.options, ...{sso_extra: options.profile.extra}};
```

Comment:
Right, I see that you introduce a new property named `sso_extra` in the UserOptions.

I suggest the following:
1. To be consistent with the other property names, have it in camel case, i.e. `ssoExtra`
2. Maybe explicit in the name that the property has no other purpose than being informative, and may not alter the behavior of the app, maybe something like: `ssoExtraInfo`
3. the second spread operator is probably not necessary, you may just do the following

```suggestion
        user.options = {...user.options, ssoExtraInfo: options.profile.extra};
```
4. And finally, add explicitly this optional property in the `UserOptions` interface, so this property can be accessed by other parts of the code (and maybe it is required for the change above)

Reply:
Thank you ! I'm taking this feedback into account. As for 4., I believe it's the case already :)